haskell/test/Test/Ligo/BaseDAO/Token/Transfer.hs

```diff
@@ -110,34 +103,11 @@ transferTests = testGroup "Transfer:"
             transfer 1 unfrozenTokens nonexistent owner1 dao
               & expectCustomError #fA2_INSUFFICIENT_BALANCE dao (#required .! 1, #present .! 0)
 
-      , nettestScenarioCaps "cannot transfer frozen tokens" $ do
-          ((owner1, _), (owner2, _), dao, _, _) <- originateWithCustomToken
-
-          withSender (AddressResolved owner1) $
-            transfer 10 frozenTokens owner1 owner2 dao
-              & expectCustomErrorNoArg #fROZEN_TOKEN_NOT_TRANSFERABLE dao
-
       , nettestScenarioCaps "fails if trying to transfer unknown tokens" $ do
           ((owner1, _), (owner2, _), dao, _, _) <- originateWithCustomToken
 
           withSender (AddressResolved owner1) $
             transfer 0 unknownTokens owner1 owner2 dao
               & expectCustomError_ #fA2_TOKEN_UNDEFINED dao
       ]
-
-  , testGroup "Transfers by the administrator"
-      [ nettestScenarioCaps "admin cannot transfer unfrozen tokens that don't belong to them" $ do
-          ((owner1, _), (owner2, _), dao, _, admin) <- originateWithCustomToken
-
-          withSender (AddressResolved admin) $
-            transfer 10 unfrozenTokens owner2 owner1 dao
-              & expectCustomError_ #fA2_NOT_OPERATOR dao
-
-      , nettestScenarioCaps "admin can transfer frozen tokens from any address to any address" $ do
-          ((owner1, _), (owner2, _), dao, _, admin) <- originateWithCustomToken
-
-          withSender (AddressResolved admin) $
-            transfer 10 frozenTokens owner1 owner2 dao
-          assertBalanceOf owner2 110 frozenTokens dao
-      ]
```

Comment:
I would keep both of these, except the second one needs to be modified to check that not even the `admin` can transfer frozen tokens from other addresses and his call fails with `FROZEN_TOKEN_NOT_TRANSFERABLE` too.